kubetest/kops.go

```diff
@@ -188,7 +188,7 @@ func (k kops) Up() error {
 	// TODO(zmerlynn): More cluster validation. This should perhaps be
 	// added to kops and not here, but this is a fine place to loop
 	// for now.
-	return waitForNodes(k, k.nodes+1, *kopsUpTimeout)
+	return waitForReadyNodes(k.nodes+1, *kopsUpTimeout)
```

Comment:
Any reason we want to use waitForNodes with kubernetes-anywhere? Can we just delete the previous logic?

Reply:
I do think we should change waitForNodes -> waitForReadyNodes everywhere, I just wanted to start with kops.  I can add a `TODO(justinsb)` here.

Reply:
I'd vote for just making the change everywhere now. We don't need backwards compat for ci :)

IMO it is more important to kill dead code